setup.cfg

```diff
@@ -1,8 +1,39 @@
-[bdist_wheel]
-universal = 1
-
 [metadata]
-license_file = LICENSE
+name = django-ecsmanage
+description = Run any Django management command on an AWS Elastic Container Service (ECS) cluster.
+long_description = file: README.rst
+url = https://github.com/azavea/django-ecsmanage/
+author = Azavea, Inc.
+author_email = systems@azavea.com
+license = Apache License 2.0
+classifiers =
+    Environment :: Web Environment
+    Framework :: Django
+    Framework :: Django :: 1.11
+    Framework :: Django :: 2.2
+    Framework :: Django :: 3.0
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.5
+install_requires =
+    Django >=1.11
+    boto3 >=1.9.0
+    future-fstrings >=1.0.0
+setup_requires =
+    setuptools_scm ==3.*
 
-[aliases]
-test = pytest
+[options.extras_require]
+tests =
+    flake8 >=3.7.7
+    black; python_version > "3.6"
```

Comment:
I'm interested to know more about what this line does. We're building 3.5, but this suggests that a component of our application is only compatible with 3.6+.

@hectcastro do you have more context?

Reply:
If somebody chooses to install the optional test dependencies (`pip install -e ".[tests]"`) this will install `black` only if they're using Python 3.7 or greater. It's excluded from the standard install.

This is relevant in `scripts/update` but we don't actually rely on `extras_require` in the Travis / `tox` build, and we aren't actually running `black` as part of the Travis build. Could be a good idea to do that, I've created #17 to track it.

Reply:
I think that this constraint was added to appease the Python version constraints defined by Black itself:

> It requires Python 3.6.0+ to run but you can reformat Python 2 code with it, too.